chrome_manifest_v3/views/popupView.js

```
@@ -427,12 +427,12 @@ Gmail2Trello.PopupView.prototype.bindEvents = function () {
   $board.off('change').on('change', () => {
     let boardId = $board.val();
 
-      let $list = $('#g2tList', self.$popup);
-  let $card = $('#g2tCard', self.$popup);
-  let $labels = $('#g2tLabels', self.$popup);
-  let $members = $('#g2tMembers', self.$popup);
-  let $labelsMsg = $('#g2tLabelsMsg', self.$popup);
-  let $membersMsg = $('#g2tMembersMsg', self.$popup);
+    let $list = $('#g2tList', self.$popup);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'$list' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)